src/modules/helpchan.ts

```diff
@@ -33,16 +33,15 @@ export class HelpChanModule extends Module {
 	CHANNEL_PREFIX = 'help-';
 
 	AVAILABLE_EMBED = new MessageEmbed()
-		.setColor(TS_BLUE)
+		.setColor('#77b155')
 		.setDescription(
-			'This help channel is now **available**, which means that ' +
-				'you can claim it by typing your question into it. ' +
-				'Once claimed, the channel will move into the **Help: Ongoing** category, and ' +
-				`will be yours until it has been inactive for ${
-					dormantChannelTimeout / 60 / 60
-				} hours or is closed ` +
-				'manually with `!close`. When that happens, it will be set to **dormant** and moved into the **Help: Dormant** category.\n\n' +
-				"Try to write the best question you can by providing a detailed description and telling us what you've tried already.",
+			'✅ **Send your question here to claim the channel**\n' +
```

Comment:
Replacing with a template string would make it easier for future people to edit.

Reply:
Done